core/src/test/java/de/bwaldvogel/mongo/backend/AbstractMongoCollectionTest.java

```diff
@@ -48,10 +69,11 @@ protected Object findDocumentPosition(Document document) {
             throw new UnsupportedOperationException();
         }
 
+        // todo: actually enforce all the parameters
```

Comment:
I realize this isn't the absolute greatest solution, but it works for the tests that are here so far. I figured it can be updated once tests are added that actually make use of these parameters.